tests/process_manager/views/test_partial_views.py

```diff
@@ -193,3 +196,32 @@ def test_filter_table(search, table, expected):
     from process_manager.views.partials import filter_table
 
     assert filter_table(search, table) == expected
+
+
+class HandleErrorsTest(TestCase):
```

Comment:
This test is using the Django `TestCase` class instead of being a pytest function like throughout the rest of the codebase. It should be updated to match the rest, as it is now, it's quite difficult to understand what it is checking.

Reply:
I can address these comments in my current pr.